src/libraries/System.Private.CoreLib/src/System/Globalization/CompareInfo.Icu.cs

```diff
@@ -692,6 +692,12 @@ private unsafe SortKey IcuCreateSortKey(string source, CompareOptions options)
             Debug.Assert(!GlobalizationMode.Invariant);
             Debug.Assert(!GlobalizationMode.UseNls);
 
+#if TARGET_BROWSER
+            // JS cannot create locale-sensitive sort key, use invaraint functions instead.
```

Comment:
could you please explain the consequences ?

Reply:
I will update the description, one of the consequences is quite severe: https://github.com/dotnet/runtime/issues/96400.